runtime/ast/composite.go

```diff
@@ -37,6 +37,7 @@ type CompositeLikeDeclaration interface {
 	Statement
 	Kind() common.CompositeKind
 	ConformanceList() []*NominalType
+	IsInterface() bool
```

Comment:
This seems to be added because with `InterfaceDeclaration` gaining `func ConformanceList() []*NominalType`, it suddenly conforms to `CompositeLikeDeclaration`.

OK if I push up a commit that removes this function and improves the AST interface conformance?

Reply:
yeah, please do!

Reply:
Pushed up 54e0ce34a714f0aef4d02408d7a8cd01b48108fc